core-web/libs/dotcms-scss/angular/dotcms-theme/components/_tree.scss

```diff
@@ -111,7 +111,7 @@
 }
 
 .p-tree .p-tree-loading-icon {
-    font-size: $font-size-xxxl;
+    font-size: $font-size-xxxxl;
```

Comment:
Do we check this value with @melissarojas-dotcms ? Looks like we have some particular fonts.

If the size is for icons, we should have custom sizes for them, such as: `$font-icon-sm`, `$font-icon-md`, `$font-icon-lg`, etc...